addons/account/models/company.py

```diff
@@ -555,3 +563,25 @@ def get_chart_of_accounts_or_fail(self):
                 "Please go to Account Configuration and select or install a fiscal localization.")
             raise RedirectWarning(msg, action.id, _("Go to the configuration panel"))
         return account
+
+    def _create_secure_sequence(self, sequence_fields):
+        """This function creates a no_gap sequence on each companies in self that will ensure
+        a unique number is given to all posted account.move in such a way that we can always
+        find the previous move of a journal entry.
+        """
+        for company in self:
+            vals_write = {}
+            for seq_field in sequence_fields:
+                if not company[seq_field]:
+                    vals = {
+                        'name': 'Securisation of ' + seq_field + ' - ' + company.name,
```

Comment:
`'name': _('Securisation of %s - %s') % (seq_field,company.name) ,`
for translation